<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/png" href="favicon-16.png" />
<title>Suprateek Chatterjee</title>
</head>	
	
<!-- MathJax -->
<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
	  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
<!-- End MathJax -->

<style type="text/css">
.abs {
 background-color: #ffffee;
 max-width: 1000px;
 padding: 7px;
 display: none;
}
li{
    margin-top: 10px;
}
li:first-child {
    margin-top:0;
}
</style>
<script type="text/javascript">
function showAbstract(e){
   f = e;
   var div;
   for(div = e.nextSibling; div.className != "abs"; div = div.nextSibling);
   if (div.style.display=="block"){
     div.style.display="";
   } else {
     div.style.display="block";
   }
   return true;
}
</script>

<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]},
                "HTML-CSS": {
                    availableFonts: ["Asana-Math"],
                    preferredFont: "Asana-Math"
                }
});
</script>	
	
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">S. Chatterjee</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="cv.pdf">CV</a></div>
<div class="menu-category">Research</div>
<div class="menu-item"><a href="publications.html" class="current">Publications</a></div>
<div class="menu-category">Miscellaneous</div>
<div class="menu-item"><a href="contact.html">Contact</a></div>
</td>

<td id="layout-content">
<div id="toptitle">
<h1>Publications</h1>
</div>

<h2>Refereed Journal papers / Preprints</h2>
<ul>

<li>
<a href="javascript:void(0)" onclick="showAbstract(this)" class="title"> <b> Network strategies to study Epstein-Barr Virus associated carcinomas and potential etiological mechanisms for oncogenesis </b></a><br />
<div class="abs">
<b>Abstract:</b> Diseased conditions are a consequence of some abnormality that are associated with clinical conditions in numerous cells and tissues affecting various organs. The common role of EBV (Epstein-Barr virus) in causing infectious mononucleosis (IM) affecting B-cells and epithelial cells and the development of EBV-associated cancers has been an area of active research. Investigating such significant interactions may help discover new therapeutic targets for certain EBV-associated lymphoproliferative (Burkitt's Lymphoma and Hodgkin's Lymphoma) and non-lymphoproliferative diseases (Gastric cancer and Nasopharyngeal cancer). Based on the DisGeNET (v7.0) data set, we constructed a disease-gene network bipartite graph to identify genes that are involved in various carcinomas namely, gastric cancer (GC), nasopharyngeal cancer (NPC), Hodgkin's lymphoma (HL) and Burkitt's lymphoma (BL). Using the community detection algorithm (Louvain method), we identified communities followed by functional enrichment using over-representation analysis methodology. In this study, we identified the modular communities to explore the relation of this common causative pathogen (EBV) with different carcinomas such as GC, NPC, HL and BL. We could identify the top 10 genes as CASP10, BRAF, NFKBIA, IFNA2, GSTP1, CSF3, GATA3, UBR5, AXIN2 and POLE based on their degree of distribution. Further over-representation analysis showed that the ABL1 gene was significantly over-represented in 3 out of 9 critical biological processes. As a result, we can infer that the EBV pathogen is selective in targeting critical pathways to bring about cellular growth arrest/apoptosis and interfering with vital biological processes, including the TP53 network of genes that leads to further proliferation of damage to vital cellular activities.
</div> 
S. Chatterjee and B. S. Sanjeev,<br />
Communicated, 2022,<br />
arXiv : 2208.00471 [q-bio.MN], [q-bio.SC] 2022.<br />
[<a href= "http://arxiv.org/abs/2208.00471" >arXiv</a>] | [<a href="http://" >pdf</a>] | [<a href="https://www." >pdf-slides</a>]
</li>	
	
	
<li>
<a href="javascript:void(0)" onclick="showAbstract(this)" class="title"> <b> Network-based community detection of comorbidities and their association with SARS-CoV-2 virus during COVID-19 pathogenesis </b></a><br />
<div class="abs">
<b>Abstract:</b> Recent studies emphasized the necessity to identify key (human) biological processes and pathways targeted by the Coronaviridae family of viruses, especially SARS-CoV-2. COVID-19 caused up to 33-55% death rates in COVID-19 patients with malignant neoplasms and Alzheimer's disease. Given this scenario, we identified biological processes and pathways which are most likely affected by COVID-19. The associations between various diseases and human genes known to interact with viruses from Coronaviridae family were obtained from the IntAct COVID-19 data set annotated with DisGeNET data. We constructed the disease-gene network to identify genes that are involved in various comorbid diseased states. Communities from the disease-gene network through Louvain method were identified and functional enrichment through over-representation analysis methodology was used to discover significant biological processes and pathways shared between COVID-19 and other diseases. The IntAct COVID-19 data set comprised of 828 human genes and 10,473 diseases that together constituted nodes in the disease-gene network. Each of the 70,210 edges connects a human gene with an associated disease. The top 10 genes linked to most number of diseases were VEGFA, BCL2, CTNNB1, ALB, COX2, AGT, HLA-A, HMOX1, FGT2 and COMT. The most vulnerable group of patients thus discovered had comorbid conditions such as carcinomas, malignant neoplasms and Alzheimer's disease. Finally, we identified 37 potentially useful biological processes and pathways for improved therapies.
</div>
S. Chatterjee and B. S. Sanjeev,<br />
Under revision (<b>Life Sciences</b>), 2022,<br />
arXiv : 2205.15968 [q-bio.MN], [q-bio.SC] 2022.<br />
[<a href= "http://arxiv.org/abs/2205.15968" >arXiv</a>] | [<a href="http://" >pdf</a>] | [<a href="https://www." >pdf-slides</a>]
</li>	

</ul>

<h2>Refereed Conference papers</h2>
<ul>

<li>
<a href="javascript:void(0)" onclick="showAbstract(this)" class="title"> <b> Identification of Human Proteins vulnerble to multiple Organisms</b></a>,<br />
<div class="abs">
<b>Abstract:</b> While most studies emphasize on certain aspects of Pathogen-Host Interactions (PHI), such as the preferential attachment of bacteria or virus to its human receptor homolog, studies have attempted to methodically classify interactions among pathogenic proteins and their host proteins. Here we have analyzed 182 pathogens from The Pathogen-Host Interaction Search Tool (PHISTO) and could identify the proteins/protein coding genes that act on both virus and bacteria. Importantly there were few proteins viz. P53 (Tumor protein p53), NF$\kappa$B1 (Nuclear factor of kappa light polypeptide gene enhancer in B-cells 1), GBLP (Guanine nucleotide-binding protein subunit beta-2-like-1), TOX4 (TOX high mobility group box family member 4), PDIA1 (Protein disulfide-isomerase precursor), MHY9 (Myosin 9), RAC1 (Ras-related C3 botulinum toxin substrate 1), CCAR2 (Cell cycle and apoptosis regulator protein 2) and ILF3 (Interleukin enhancer binding factor 3) that were more susceptible to both bacterial and viral pathogens. Identification of such important interacting proteins (IIPs) can elicit significant insights for better understanding the molecular mechanisms of such pathogens that interact with the human host.
</div>
S. Chatterjee and B. S. Sanjeev,<br />
In Proceedings of International Conference on Bioinformatics and Systems Biology (<b>BSB</b>), pp. 1-4, March 04-06, 2016. [<a href="http://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=7548213" >Link</a>]<br />
arXiv : 1609.03337 [q-bio.MN], 2016.<br />
[<a href="https://doi.org/10.1109/BSB.2016.7552164" >DOI</a>] | [<a href= "http://arxiv.org/abs/1609.03337" >arXiv</a>] | [<a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7552164" >pdf</a>] | [<a href="docs/" >pdf-slides</a>] | [<a href="docs/P1.pdf" >poster</a>] | [BibTeX] </a> <br />
</li>

</ul>

<h2>Thesis</h2>
<ul>

<li>
<a href="javascript:void(0)" onclick="showAbstract(this)" class="title"> <b> A Contact Map-Based Approach To Predict Conformational Epitope Residues </b></a><br />
<div class="abs">
<b>Abstract:</b> We apply contact mapping techniques once we have successfully classified the exposed residues from the buried ones using some cut-off distance $\delta$, which are further grouped into clusters of exposed residues that fall under the threshold cut-off distance in contact map space.
</div>
S. Chatterjee <br />
M.Tech, IIIT-Allahabad <br />	
[<a href="docs/" >pdf-slides</a>]</a> <br />
</li>

</ul>	
	
<div id="footer">
<div id="footer-text">Page generated on 22-08-2014 at 23:59:59 IST.
</div>
</div>
</td>
</tr>
</table>

<!-- Google Analytics 

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-56996692-2', 'auto');
ga('send', 'pageview');
</script>

End Google Analytics -->

</body>

</html>
